public/css/list/downtime-list.less

```diff
@@ -24,9 +24,15 @@
     }
   }
 
-  .visual.active {
-    background-color: @color-ok;
-    color: @text-color-on-icinga-blue;
+  &.active {
+    .visual {
+      background-color: @color-ok;
+      color: @text-color-on-icinga-blue;
+    }
+
+    .main {
+      padding-top: 0; // If active the progress bar represents the padding top
```

Comment:
I'm not sure, why you did this. But sadly this has the following effect.

![Screen Recording 2021-09-27 at 14 27 58](https://user-images.githubusercontent.com/6977434/134908230-21e16399-cae5-4d75-b334-da71e84ed588.gif)

The `.active` class is for selected `.list-item`s not active downtime items

Reply:
Yeesss, it collides with the action list behavior. There `active` is also the class used to indicate which list item is active :roll_eyes: